<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newHostForm</class>
 <widget class="QDialog" name="newHostForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Groupe</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="addGroupHost">
                <property name="text">
                 <string>ADD</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="disableGroupHost">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>DISABLE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteGroupHost">
                <property name="text">
                 <string>DELETE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxGroup"/>
            </item>
            <item>
             <widget class="QListWidget" name="listGroupHost"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Hosts</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="addHost">
                <property name="text">
                 <string>ADD</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="disableHost">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>DISABLE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteHost">
                <property name="text">
                 <string>DELETE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxHosts"/>
            </item>
            <item>
             <widget class="QListWidget" name="listHosts"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newHostForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newHostForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
